app/src/main/java/com/lastaoutdoor/lasta/ui/screen/discover/DiscoverScreen.kt

```diff
@@ -445,3 +449,22 @@ fun ActivitiesDisplay(
         }
   }
 }
+
+@Composable
+fun EmptyActivityList() {
+  Column(
+      modifier = Modifier.fillMaxSize().testTag("EmptyActivityList"),
+      verticalArrangement = Arrangement.Center,
+      horizontalAlignment = Alignment.CenterHorizontally) {
+        Icon(
+            painterResource(id = R.drawable.not_found),
+            contentDescription = "No Activities",
+            modifier = Modifier.size(85.dp).testTag("NoActivitiesLogo"))
+        Text(
+            text = LocalContext.current.getString(R.string.no_activities),
+            style =
+                TextStyle(
+                    fontWeight = FontWeight.Normal, fontSize = 20.sp, textAlign = TextAlign.Center),
```

Comment:
For the text, is better to use MaterialTheme.typography package for different sort of texts so that it looks homegenous through out the app! Hardcoded text fonts wont adapted themselves with the screen size which is an UI issue